build.gradle.kts

```diff
@@ -105,6 +123,7 @@ tasks {
 
     withType<KotlinCompile> {
         kotlinOptions.jvmTarget = "12"
```

Comment:
Hvis det er enkelt å oppgradere til java 13 hadde det vært supert, for da kan jeg bygge med riktig java-versjon hos meg også 😅 Testcontainers funker jo ikke med java 12 hos meg.

Reply:
jeg oppdaterer :)